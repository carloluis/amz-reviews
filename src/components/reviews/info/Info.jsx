import React from 'react';
import PropTypes from 'prop-types';
import styles from './Info.scss';

const Info = ({ title, text }) => (
    <div className={styles.container}>
        <div className={styles.title}>{title}</div>
        <div className={styles.text}>{text}</div>
    </div>
);

Info.propTypes = {
    title: PropTypes.string.isRequired,
    text: PropTypes.string.isRequired
};

export default Info;
