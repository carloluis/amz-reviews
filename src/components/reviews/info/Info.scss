@import '../../../styles/palette.scss';

.container {
    font-weight: 600;
    max-width: 140px;
    line-height: 1.4;

    .title {
        text-transform: capitalize;
        color: $light-font;
    }

    .text {
        color: $dark-font;
        overflow: hidden;
        white-space: nowrap;
        text-overflow: ellipsis;
    }
}
