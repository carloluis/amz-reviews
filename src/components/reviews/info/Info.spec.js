import React from 'react';
import renderer from 'react-test-renderer';
import Info from './Info';

describe('<Info/>', () => {
    it('snapshot', () => {
        const result = renderer.create(<Info title="Info Title" text="Info Description" />);
        expect(result).toMatchSnapshot();
    });
});
