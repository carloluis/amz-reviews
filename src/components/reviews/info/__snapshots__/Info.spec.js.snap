// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Info/> snapshot 1`] = `
<div
  className="container"
>
  <div
    className="title"
  >
    Info Title
  </div>
  <div
    className="text"
  >
    Info Description
  </div>
</div>
`;
